apps/newsletters-api/src/main.ts

```diff
@@ -14,11 +15,42 @@ app.get('/health', async () => {
 	});
 });
 
+// not using the makeSuccess function on this route as
+// we are emulating the response of the legacy API
 app.get('/v1/newsletters', async (req, res) => {
-	const parsedLive = liveNewslettersData.filter(isNewsletter);
-	return parsedLive;
+	// TO DO - why is eslint not happy here?
```

Comment:
The comment was about an "Unsafe assignment of an `any` value" warning being flagged on line 22, but that seems to have gone now. It might have been a VScode glitch that just needed a restart. I have found that big  Zod types seem to strain VScode a little.
I'll remove the comment.